deploy/devel/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ kind: Kustomization
 commonLabels:
   app.kubernetes.io/part-of: toolforge-build-service
 bases:
+  - ../base-tekton
```

Comment:
I had problems in the past where base-tekton needed to be done before base, otherwise there was a race condition and the deploy ended in failure, from the docs:
```
This two step deploy is a limitation of the current deployment system, as you can't use the CRDs in the same run you create them.
```

Did you test this?

An alternative is running the `kubectl apply -k  ...` twice in the deploy.sh script, so both steps don't execute under the same kompose run

Reply:
Yeaaa I wanted to mention this after I made the pull request but forgot. I tested it several times and didn't notice any issues. Any way we can reproduce the issue you mentioned?

Reply:
Hmm, I tested it a few times too (destroying the cluster and creating it anew) using both my os installed kubectl (v1.20.2) and the one with minikube (v1.21.8) with no issues, I think that we can try deploying with this on toolsbeta and if it works call it a win xd

Maybe keep a comment for now in case it fails at some other point, saying something like `in the past we have seen kustomize failing to apply the crds at the same time that resources that use them, if that happens, try deploying the base-tekton first`, wdyt?

Reply:
Yea this will do